client/src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -249,7 +246,7 @@ private synchronized PlatformError startTunnel(
           "failed to connect the tunnel");
     }
     startForegroundWithNotification(config.name);
-    storeActiveTunnel(config, remoteUdpForwardingEnabled);
```

Comment:
This is where we store the UDP connectivity state for resuming a connection. Do we need to store it somewhere in the new arch?

Reply:
I was thinking we should check it again. I don't think it makes sense to save this, since we check on network changes.